### ISSUE TITLE
Option to use fastq-dump --split-files,  instead of pyfilesplit

### ISSUE THREAD

**Issue by mimakaev** [open2c/distiller-nf#155]
I noticed that some of the .sra files are missing one side for a large fraction of the reads. Those are usually due to some problems with processing, but it may be nice to be able to map them as is. It would be hence nice to have an option to download/split files using fastq-dump --split-files. 

https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR8453396  is an example of an SRR that is missing 50% of one of the two sides. pyfilesplit does not handle this perfectly.

**Comment by golobor** [comment]
i just ran into the same issue! Can you expand a little bit on this - why do you think that fastq-dump would handle this better?

**Comment by golobor** [comment]
ok, i'll implement an option to use fastq-dump --split-3 then?..

**Comment by mimakaev** [comment]
I think it should be "--split-files --gzip"   

Historically we chose not to do this and wrote _pyfilesplit_ for the following reason. _Fastq-dump_ is fast intrinsically, but if you ask it to gzip the output, it does so using the same thread and one core, and ends up painfully slow. Also, there was no nice way to make _fastq-dump_ write to two streams, so we wrote _pyfilesplit_  to make splitting for us. 

This all makes sense only if downstream processing (pairtools) is fine with the read being on one side, and not on the other side... @golobor   Is this true?
